apps/studio/src/features/dashboard/components/FolderSettingsModal/FolderSettingsModal.tsx

```diff
@@ -95,12 +95,16 @@ const SuspendableModalContent = ({
   const { mutate, isLoading } = trpc.folder.editFolder.useMutation({
     onSettled: onClose,
     onSuccess: async () => {
-      await utils.site.list.invalidate()
       await utils.resource.listWithoutRoot.invalidate()
       await utils.resource.getChildrenOf.invalidate({
         resourceId: parentId ? String(parentId) : null,
       })
-      await utils.folder.getMetadata.invalidate()
+      await utils.folder.getMetadata.invalidate({
```

Comment:
Nit: We can probably invalidate everything; no big deal for things to fetch again. Invalidation doesn’t mean it triggers a refetch immediately anyways